### PR TITLE
Maria resources

### PR DESCRIPTION
**Pull request by MariaShaposhnikova**

### CODE REVIEW

File: content/refguide/resources.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Resources"
```
Comment (MarkvanMents) [inline content/refguide/resources.md]: The issue here is that Studio Pro calls these things *Documents*. Perhaps we can look at this in the meeting?

Comment (MarkvanMents) [reply content/refguide/resources.md]: image

Comment (MariaShaposhnikova) [reply content/refguide/resources.md]: In UI they are also called "resources" when you add them.

Comment (MarkvanMents) [reply content/refguide/resources.md]: No - you are right - they can be called resources, or perhaps resource documents?
perhaps put a picture to show what you mean?

**Pull request merged**